大規模ソフトウェア論第6回課題

pk-Musan opened the pull request:
大規模ソフトウェア論第6回の課題「UC3：つぶやきを検索する の実装」の提出になります。
ご確認のほどよろしくお願い致します。

pk-Musan committed:
ユースケース2まで

pk-Musan committed:
UC3：つぶやきを検索するの実装

pk-Musan committed:
検索をGetMappingに変更(実装には特に変更なし)

muretti0114 commented:
良くできています．ほぼ完璧です．マージしてください．

- サービスクラスのpublic List<Tsubuyaki> searchTsubuyaki( String serachWord )
引数のスペルが（^^;）．．．

pk-Musan committed:
Update TsubuyakiService.java

searchTsubuyakiメソッドの引数のタイプミスを修正
38: serachWord -> searchWord
39: serachWord -> searchWord
